src/openwith.js

```diff
@@ -16,21 +16,31 @@
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 //
 OME.setOpenWithUrlProvider("omero_iviewer", function(selected, url) {
+    selected_count = selected.length;
+    // last selected image
+    last_selected = selected[selected_count-1];
     // Add image Id to url
-    url += selected[0].id + "/";
+    url += last_selected.id + "/";
```

Comment:
This seems to cause that when I multiselect couple of images in webclient (either with Shift + click or via drag in central pane thumbs) I alwasy end with the order of  (assuming for example 4 selected images):
 1. the last (fourth) selected image
 2. the first selected image
 3. the second selected image
 4. the one-but-last (third) selected image

This ordering makes no sense, because it is impossible to compare the order with webclient easily. The order should be:
1. the first selected image
2. the second selected image
3. the third selected image
4. the last (fourth) selected image